wear/src/main/java/io/homeassistant/companion/android/home/views/SensorsView.kt

```diff
@@ -0,0 +1,99 @@
+package io.homeassistant.companion.android.home.views
+
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.Chip
+import androidx.wear.compose.material.ChipDefaults
+import androidx.wear.compose.material.ExperimentalWearMaterialApi
+import androidx.wear.compose.material.PositionIndicator
+import androidx.wear.compose.material.Scaffold
+import androidx.wear.compose.material.ScalingLazyColumn
+import androidx.wear.compose.material.ScalingLazyListState
+import androidx.wear.compose.material.Text
+import androidx.wear.compose.material.rememberScalingLazyListState
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.sensors.SensorReceiver
+import io.homeassistant.companion.android.theme.WearAppTheme
+import io.homeassistant.companion.android.common.R as commonR
+
+@ExperimentalAnimationApi
+@ExperimentalWearMaterialApi
+@Composable
+fun SensorsView(
+    onClickSensorManager: (SensorManager) -> Unit
+) {
+    val scalingLazyListState: ScalingLazyListState = rememberScalingLazyListState()
+
+    WearAppTheme {
+        Scaffold(
+            positionIndicator = {
+                if (scalingLazyListState.isScrollInProgress)
+                    PositionIndicator(scalingLazyListState = scalingLazyListState)
+            },
+            timeText = { TimeText(!scalingLazyListState.isScrollInProgress) }
+        ) {
+            val sensorManagers = getSensorManagers()
+            ScalingLazyColumn(
+                modifier = Modifier
+                    .fillMaxSize(),
```

Comment:
I think we need to add the rotary input handler here so we can scroll with the bezel like we can in other screens. Eventually we wont need to add this to every screen but I am not sure when Google will add the native functionality :)

As of now it would need to be added on each new page we introduce, just to keep things consistent.